### PR TITLE
AP-4143 partner asset questions

### PR DESCRIPTION
**Pull request by RoseSAK**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4143)

Update the following pages for the partner means journey:
* Savings and investments
* Other assets
* Restrictions
* Payments from schemes or charities

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: config/locales/en/providers.yml [hunk 1]
```diff
@@ -1103,15 +1103,17 @@ en:
...
+            partner: Which schemes or trusts have paid either your client or their partner?
           england_infected_blood_support: England Infected Blood Support Scheme
           vaccine_damage_payments: Vaccine Damage Payments Scheme
           variant_creutzfeldt_jakob_disease: Variant Creutzfeldt-Jakob disease (vCJD) Trust
           criminal_injuries_compensation_scheme: Criminal Injuries Compensation Scheme
           national_emergencies_trust: National Emergencies Trust (NET)
           we_love_manchester_emergency_fund: We Love Manchester Emergency Fund
           london_emergencies_trust: The London Emergencies Trust
-          none_of_these: My client has received none of these payments
+          none_of_these: None of these payments
```
Comment (colinbruce) [inline config/locales/en/providers.yml]: I thought content wanted these moved _away_ from none of the above to be clearer! 🤔

**Review by colinbruce — APPROVED** [review]
Looks okay, apart from one non-blocker about content, :+1:

File: app/views/providers/means/other_assets/show.html.erb [hunk 2]
```diff
@@ -2,21 +2,47 @@
...
-                     collection: Citizens::OtherAssetsForm::SINGLE_VALUE_ATTRIBUTES,
-                     locals: { model: @form, form: } %>
+          <%= form.govuk_check_box :check_box_valuable_items_value,
+                                   true,
+                                   multiple: false,
+                                   link_errors: true,
+                                   label: { text: t(".check_box_valuable_items_value") },
+                                   hint: { text: t(".hint.check_box_valuable_items_value", default: "") } do %>
+            <% value = @form.send(:valuable_items_value) %>
+            <% @form.__send__(:check_box_valuable_items_value=, @form.send(:check_box_valuable_items_value).present? || value.present?) %>
```
[the remark this answers is not in the document]
Comment (colinbruce) [reply app/views/providers/means/other_assets/show.html.erb:20]: re-reading, I think it's fine but feels a bit icky to be doing this in the view!

[the remark this answers is not in the document]
Comment (RoseSAK) [reply app/views/providers/means/other_assets/show.html.erb:20]: Yeah, it looks horrible :( but without it the checkbox value wasn't retained when the page reloaded e.g. for an error message. There might be some other issue or way of doing it that's better though/doesn't look as bad

[the remark this answers is not in the document]
Comment (RoseSAK) [reply app/views/providers/means/other_assets/show.html.erb:20]: There are issues on most of these pages with error messages not linking to the fieldset properly, but I've created another ticket to fix it as this ticket was dragging on and also it was an existing bug so not directly related to the changes in this ticket.

[the remark this answers is not in the document]
Comment (RoseSAK) [reply app/views/providers/means/other_assets/show.html.erb:20]: https://dsdmoj.atlassian.net/browse/AP-4504

[the remark this answers is not in the document]
Comment (colinbruce) [reply app/views/providers/means/other_assets/show.html.erb:20]: Have discussed - happy for this to be moved to UAT and we can address after 4141/other work is complete

**Pull request merged**